components/alert/style/index.less

```diff
@@ -111,6 +111,11 @@
     font-size: @font-size-base;
   }
 
+  &-message {
```

Comment:
could you place it on line `86` above `&-with-description` to not interrupt the `&-with-description` block?

Reply:
Done!

Reply:
although the overflow still happens if the element is wider than `16px` not sure if `padding` is the correct solution here:
![screen shot 2018-11-20 at 17 29 53](https://user-images.githubusercontent.com/9677898/48788090-5f4a3000-ecea-11e8-9f87-c0964d5b228e.png)

Reply:
You're right. I made a new commit using float right instead and that seems to fix it.

Reply:
rebase and update snapshops

Reply:
I'm clueless as to how to update snapshots; do I do that by hand or is there some sort of command? Thanks!

Reply:
`yarn test -u` || `npm test -- -u`

Reply:
now `&-message` is fixed, but `<Alert description="Warning Text Warning Text Warning TextW arning Text Warning Text Warning TextWarning Text" />` still have the issue 

<img width="625" alt="screenshot 2018-11-21 at 21 54 19" src="https://user-images.githubusercontent.com/9677898/48867952-3d2edb80-edd8-11e8-8494-8d1ea9eb941d.png">